packages/service-library/src/servicelib/rest_constants.py

```diff
@@ -16,3 +16,6 @@ class PydanticExportParametersDict(TypedDict):
     exclude_defaults=False,
     exclude_none=False,
 )
+
+# Headers keys
+X_PRODUCT_NAME_HEADER: Final[str] = "X-Simcore-Products-Name"
```

Comment:
minor: isn't there some singular vs plural issue here?

Reply:
I agree. But I just moved this one from the webserver to here, so I don't want to mess with it. That would surely cause issues